src/treemap/index.js

```diff
@@ -132,10 +135,24 @@ class Treemap extends React.Component {
     return [];
   }
 
+  /**
+   * Triggers a callback on a box if the callback is set.
+   * @param {function} handler Callback function.
+   * @param {Object} d Data point of the arc.
+   * @param {Object} event Event.
+   * @private
+   */
+  _triggerHandler(handler, d, event) {
```

Comment:
Why not setting the default value of the handlers to noop, this way you don't need to do this and the code would be cleaner